frontend/packages/console-app/src/actions/modify-hpa.ts

```diff
@@ -7,6 +8,11 @@ import {
 } from '@console/internal/module/k8s';
 import { HorizontalPodAutoscalerModel } from '@console/internal/models';
 import deleteHPAModal from '@console/dev-console/src/components/hpa/DeleteHPAModal';
+import {
+  TYPE_OPERATOR_WORKLOAD,
+  TYPE_OPERATOR_BACKED_SERVICE,
+} from '@console/dev-console/src/components/topology/operators/components/const';
+import { isOperatorBackedService } from '@console/dev-console/src/components/topology';
```

Comment:
we should not import from `@console/dev-console/*` in `console-app`

Reply:
Good catch... I apparently made this mistake on line 9 😞